packages/synapse-react-client/src/lib/containers/auth/Login.tsx

```diff
@@ -1,199 +0,0 @@
-import * as React from 'react'
```

Comment:
Replaced with LoginForm.tsx (layout) and StandaloneLoginForm.tsx (statefulness/functionality)